eth-bytecode-db/eth-bytecode-db-server/tests/compiler_versions.rs

```diff
@@ -0,0 +1,113 @@
+mod verification_test_helpers;
+
+use crate::verification_test_helpers::{
+    smart_contract_verifer_mock::{
+        MockSolidityVerifierService, MockVyperVerifierService, SmartContractVerifierServer,
+    },
+    VerifierService,
+};
+use eth_bytecode_db_proto::blockscout::eth_bytecode_db::v2 as eth_bytecode_db_v2;
+use pretty_assertions::assert_eq;
+use rstest::rstest;
+use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v2 as smart_contract_verifier_v2;
+use tonic::Response;
+
+async fn test_versions(
+    route: &str,
+    verifier: impl VerifierService<smart_contract_verifier_v2::ListCompilerVersionsResponse>,
+    verifier_response: smart_contract_verifier_v2::ListCompilerVersionsResponse,
+) {
+    let db_url = "sqlite::memory:";
+    let verifier_addr =
+        verification_test_helpers::init_verifier_server(verifier, verifier_response.clone()).await;
+    let eth_bytecode_db_base =
+        verification_test_helpers::init_eth_bytecode_db_server(db_url, verifier_addr).await;
+
+    let response = reqwest::Client::new()
+        .get(eth_bytecode_db_base.join(route).unwrap())
+        .send()
+        .await
+        .expect("Failed to send request");
+
+    // Assert that status code is success
+    if !response.status().is_success() {
+        let status = response.status();
+        let message = response.text().await.expect("Read body as text");
+        panic!(
+            "Invalid status code (success expected). Status: {}. Message: {}",
+            status, message
+        )
+    }
+
+    let response: eth_bytecode_db_v2::ListCompilerVersionsResponse = response
+        .json()
+        .await
+        .expect("Response deserialization failed");
+
+    assert_eq!(
+        verifier_response.compiler_versions, response.compiler_versions,
+        "Compiler versions mismatch"
+    );
+}
+
+#[rstest]
+#[tokio::test]
+#[timeout(std::time::Duration::from_secs(60))]
```

Comment:
is there a reason to add explicit timeout here? you can just run all tests with assert https://stackoverflow.com/questions/73131115/unit-test-timeout-in-rust